app/models/dataset/DatasetService.scala

```diff
@@ -77,8 +77,15 @@ class DatasetService @Inject()(organizationDAO: OrganizationDAO,
   ): Fox[Dataset] = {
     implicit val ctx: DBAccessContext = GlobalAccessContext
     val newId = ObjectId.generate
-    val details =
-      Json.obj("species" -> "species name", "brainRegion" -> "brain region", "acquisition" -> "acquisition method")
+    val metadata =
+      if (publication.isDefined)
+        Json.arr(
+          Json.obj("type" -> "string", "key" -> "species", "value" -> "species name", "index" -> 0),
```

Comment:
Why is the "index" needed? If the data model is an array, the ordering should already be fixed, right?

Reply:
> I could remove the index and always re-add it in the frontend 🤔

Its now implemented that way